sql-cloudant/src/main/scala/org/apache/bahir/cloudant/CloudantConfig.scala

```diff
@@ -95,15 +189,49 @@ class CloudantConfig(val protocol: String, val host: String,
     }
   }
 
+  def getTotalDocCount: Int = {
+    val limit = 1
+    if (viewPath != null) {
+      // "limit=" + limit + "&skip=" + skip
+      buildViewRequest(limit, includeDocs = false).build().getResponse.getTotalRowCount.toInt
+    } else {
+      // /_all_docs?limit=1
+      // Note: java-cloudant's AllDocsRequest doesn't have a getTotalRowCount method
+      // buildAllDocsRequest(1, includeDocs = false).build().getResponse.getTotalRowCount.toInt
+      val response = client.executeRequest(Http.GET(
+        new URL(database.getDBUri + File.separator + endpoint + "?limit=" + limit)))
+      getResultTotalRows(response.responseAsString)
```

Comment:
It might be easier to use
`com.cloudant.client.api.model.DbInfo#getDocCount` instead of trying to do it via `_all_docs`